sdk/search/test-resources.bicep

```diff
@@ -0,0 +1,21 @@
+param baseName string = resourceGroup().name
+param searchEndpointSuffix string = 'search.windows.net'
+
+var apiVersion = '2024-06-01-preview'
+
+resource searchService 'Microsoft.Search/searchServices@2024-06-01-preview' = {
```

Comment:
Do we want to use string interpolation here?

Reply:
You would get the following error: "String interpolation is unsupported for specifying the resource type.bicep(BCP047)"